website/docs/cloud-docs/api-docs/no-code-provisioning.mdx

```diff
@@ -0,0 +1,491 @@
+      "self": "/api/v2/no-code-modules/nocode-9HE91XDNY3faePn2"
+    }
+  }
+}
+```
+
+## Update No-Code Provisioning settings for a module.
+
+`PATCH /no-code-modules/:id`
+
+Parameter | Description
+--- | ---
+`:id` | The unique identifier for a No-Code-Allowed module.
+
+Use this API to update the settings for No-Code Provisioning of a module. You can use this API to:
+
+- Enable or disable No-Code Provisioning using this module
+- Make adjustments to the set of allowed variable values
+- Change the version of the module used during provisioning
+- Change the module that is used during provisioning
+
+The unique identifier for the No-Code-Allowed module was returned in the response to [the API call that initially enabled the module for No-Code Provisioning](#allow-no-code-provisioning-of-a-module-for-an organization).
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](/terraform/cloud-docs/users-teams-organizations/api-tokens#organization-api-tokens). You must access it with a [user token](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
+
+| Status  | Response                                     | Reason                                                         |
+| ------- | -------------------------------------------- | -------------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "no-code-modules"`) | Successfully updated a no-code module                                    |
+| [404][] | [JSON API error object][]                    | Not found, or user unauthorized to perform action |
+| [422][] | [JSON API error object][]                    | Malformed request body (missing attributes, wrong types, etc.) |
+| [500][] | [JSON API error object][]                    | Internal system failure                                   |
+
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+Key path | Type | Default | Description
+--- | --- | --- | ---
+`data.type` | string | | Must be `"no-code-modules"`.
+`data.attributes.version-pin` | string   | (previous value) | The version of the module that should be used in No-Code Provisioning workflows.
+`data.attributes.enabled` | boolean   | (previous value) | Set to `true` in order to enable No-Code Provisioning workflows of `false` to disable.
+`data.relationships.registry-module.data.id` | string | (previous value) | The ID of a module in the organization's Private Registry.
+`data.relationships.registry-module.data.type` | string | (previous value) | Must be `"registry-module"`.
+`data.relationships.variable-options.data[].id` | string | (new value) | The ID of an existing variable-options set. If set, the existing options with this ID will be replaced. If unset, a new variable-option set will be created.
```

Comment:
I found "If set" a little unclear, could we use "if provided" instead to be even more explicit?
```suggestion
`data.relationships.variable-options.data[].id` | string | (new value) | The ID of an existing variable-options set. If provided, a new variable-options set replaces the set with this ID. If not provided, this creates a new variable-option set.
```